prime-router/build.gradle.kts

```diff
@@ -900,7 +900,9 @@ dependencies {
     implementation("ca.uhn.hapi.fhir:org.hl7.fhir.r4:6.3.24")
     implementation("ca.uhn.hapi:hapi-base:2.5.1")
     implementation("ca.uhn.hapi:hapi-structures-v251:2.5.1")
+    implementation("ca.uhn.hapi:hapi-structures-v26:2.5.1")
```

Comment:
I made a comment in `ORM_O01_PATIENT.java` about this. Basically, the 2.6 structure library is only necessary because `ORM_O01_INSURANCE.class` doesn't exist in 2.7. Once we've implemented the INS resources this can be removed.